test/e2e/utils/fixture/fixture.go

```diff
@@ -17,7 +17,7 @@ import (
 	topologyclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
 )
 
-var F Fixture
+var f Fixture
```

Comment:
as much as I love to use name as terse as possible, can we rename this `fxt` to make it more grep-friendly?

also see: https://google.github.io/styleguide/go/decisions#variable-names

Reply:
Sure. thank you always for the added value!